ts/indicators/atr.src.ts

```diff
@@ -138,15 +138,15 @@ seriesType<Highcharts.ATRIndicator>(
      */
     {
         getValues: function (
-            series: Highcharts.ATRIndicator,
+            series: Highcharts.Series,
             params: Highcharts.ATRIndicatorParamsOptions
         ): (boolean|Highcharts.IndicatorValuesObject) {
-            var period = (params.period as any),
-                xVal = (series.xData as any),
-                yVal = series.yData,
-                yValLen = yVal ? yVal.length : 0,
-                xValue = (xVal as any)[0],
-                yValue = yVal[0],
+            var period: number = (params.period as any),
+                xVal: Array<number> = (series.xData as any),
+                yVal: Array<Array<number>> = (series.yData as any),
```

Comment:
Additionally the `ATRIndicator.yData` property should be removed in the class declaration.